.travis.yml

```diff
@@ -3,3 +3,12 @@ elixir:
   - 1.6.6
   - 1.7.4
   - 1.8.2
+  - 1.9.1
+
+jobs:
+  include:
+    - stage: check style guide
```

Comment:
Can we call it `lint` instead?

Reply:
That sounds good aswell. I'll change it. :smiley: Should the readme also be changed back into something like `To run the linters`? Similar how it previously looked.

Reply:
I think the readme is fine just like that! Thanks for taking care :)

Reply:
Ok, I changed the name of the stage to lint. Thanks for the quick reply :smiley: